runtime.go

```diff
@@ -573,12 +573,8 @@ func getWorkingDir(basedir string, codeuri string) (string, error) {
 
 	dir := filepath.Join(abs, codeuri)
 
-	// ...but only if it actually exists
```

Comment:
That's fine, if os.stat fails, we will default to basedir which is the mount point passed thru CLI paramerter right? 

I think we should not remove this code. What you think?